updated requires to reflect functional changes in package

antonhibl opened the pull request:
it looks like the openai-completion--data-choices and
    openai-completion--get-choices functions were renamed to
    openai--data-choices and openai--get-choices in the openai package. I also
    changed the require to openai vs openai-collections as that is the full
    package name. addresses #5.

antonhibl committed:
updated requires to reflect functional changes in package

    it looks like the openai-completion--data-choices and
    openai-completion--get-choices functions were renamed to
    openai--data-choices and openai--get-choices in the openai package. I also
    changed the require to openai vs openai-collections as that is the full
    package name. addresses #5.

antonhibl commented:
addresses #5

jcs090218 commented on codegpt.el:
```diff
@@ -31,7 +31,7 @@
 ;;; Code:
 
-(require 'openai-completion)
+(require 'openai)
```
I think we might need both.

```elisp
(require 'openai)
(require 'openai-completion)
```

antonhibl replied:
yeah, seems like importing both should work

antonhibl replied:
just updated with both @jcs090218

jcs090218 reviewed (commented):
Thanks. I forgot to change the API here. 😅

antonhibl committed:
added second require for completions

antonhibl commented:
@jcs090218 it looks like the `(require 'openai-completions) line is throwing off the CI system, are we sure we need that line f we have `(require 'openai) above it? Importing openai should handle importing the same things I believe although correct me if I am wrong because you are the author.

antonhibl committed:
fix CI building

antonhibl commented:
let me know if you see the same CI issues, I just removed the (require 'openai-completions) line from this commit since it cause the CI to fail.

jcs090218 commented:
I think we only need `openai-completion` since it already require `openai` by itself. See https://github.com/emacs-openai/openai/blob/e1a7fa234faa1130e997a942e96fe18d9785d0f3/openai-completion.el#L29

antonhibl committed:
updated requires, typo error

    I had missed the s on the end of completions before 🤦 in `(require
    'openai-completions)`.

antonhibl committed:
typo

    completions -> completion

antonhibl commented:
Makes sense, it should be fixed now. Not sure why the CI failed before since in this case it is a valid require for openai-completion @jcs090218

jcs090218 commented:
I think you added the extra `s` behind. https://github.com/emacs-openai/codegpt/actions/runs/4158060485/jobs/7193041221#step:5:125

antonhibl commented:
that makes sense, oops 😅

jcs090218 commented:
No worries! Thanks for your contribution!

Feel free to make more contributions! Looking forward to collaborating with others! ;)
